### PR TITLE
Feature/alert

### PR DESCRIPTION
**Pull request by bhanuprakashj**
image
To check the output please refer readme file

### CODE REVIEW

**Comment by omjadhav85** [comment]
Looks good to merge...again! :p

**Comment by shuklahimanshu81** [comment]
Don't just leave a comment here. Leave a comment using review changes button above in files changed section otherwise it will be not be counted in review.

File: src/css/components/alert.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.alert {
+	padding: 1rem;
+	border-radius: 8px;
+	margin-bottom: 10px;
+	box-shadow: 1px 1px 2px var(--box-shadow-color);
+}
```
Comment (shuklahimanshu81) [inline src/css/components/alert.css]: Use rems

**Review by shuklahimanshu81 — COMMENTED** [review]
Everything else is good . Code is clean. Good work. Can be merged.

File: src/css/components/alert.css (hunk 1 above)
Comment (bhanuprakashj) [reply src/css/components/alert.css]: changed to rems

**Pull request merged**